### PR TITLE
NF- 1252 wallet photo collections viewer

### PR DESCRIPTION
**Pull request by parth-simformsolutions**
### 🔗 Issue Links

_Provide all Jira tickets and/or Github issues related to this PR, if applicable._

### 🎯 Goal

_Describe why we are making this change._

### 📝 Summary

_Provide bullet points with the most important changes in the codebase._

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

| Before | After |
| ------ | ----- |
|  img   |  img  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist
- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] This change follows zero ⚠️ policy (required)
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS)

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_

### CODE REVIEW

File: Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift [hunk 1]
```diff
@@ -624,6 +637,17 @@ open class ChatMessageListVC:
         return !message.isBlank
     }
 
+    private func isImagePreview(_ message: ChatMessage?) -> Bool {
```
Comment (jitendra-simform) [inline Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift:640]: @parth-simformsolutions  it is more convenient if you add this method to ChatMessage class inside extension , thanks

File: Sources/StreamChat/Models/ChatMessage.swift [hunk 2]
```diff
@@ -352,6 +352,16 @@ public extension ChatMessage {
         extraData.keys.contains("redPacketPickup") ?? false
     }
 
+    public var isMultiplePreview: Bool {
```
Comment (ajay-simform) [inline Sources/StreamChat/Models/ChatMessage.swift]: @parth-simformsolutions  The variable name should be descriptive.

File: Sources/StreamChatUI/ChatMessageList/Attachments/VideoAttachmentGalleryPreview.swift [hunk 3]
```diff
@@ -20,12 +20,12 @@ open class VideoAttachmentGalleryPreview: _View, ThemeProvider {
     open var didTapOnUploadingActionButton: ((ChatMessageVideoAttachment) -> Void)?
     
     /// An image view used to display video preview image
-    open private(set) lazy var imageView = UIImageView()
+    open lazy var imageView = UIImageView()
```
Comment (ajay-simform) [inline Sources/StreamChatUI/ChatMessageList/Attachments/VideoAttachmentGalleryPreview.swift]: @parth-simformsolutions  Please change to private(set)

File: Sources/StreamChatUI/ChatMessageList/ChatMessage/AttachmentPreviewBubble.swift [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+//
+
+import UIKit
+import StreamChat
+
+class AttachmentPreviewBubble: UITableViewCell {
+    var layoutOptions: ChatMessageLayoutOptions?
+    weak var delegate: PhotoCollectionAction?
+    var content: ChatMessage?
+    var isSender = false
```
Comment (ajay-simform) [inline Sources/StreamChatUI/ChatMessageList/ChatMessage/AttachmentPreviewBubble.swift]: @parth-simformsolutions  isSender is not in use

File: Sources/StreamChatUI/ChatMessageList/ChatMessage/AttachmentPreviewBubble.swift [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+                imagePreview.removeFromSuperview()
+            }
+            imagePreview = UIImageView()
+            imagePreview.backgroundColor = Appearance.default.colorPalette.background6
+            imagePreview.contentMode = .scaleAspectFill
+            imagePreview.transform = .mirrorY
+            imagePreview.clipsToBounds = true
+            imagePreview.layer.cornerRadius = 12
+            loadingIndicator.isHidden = false
+            imageTask = Components.default.imageLoader.loadImage(
```
Comment (ajay-simform) [inline Sources/StreamChatUI/ChatMessageList/ChatMessage/AttachmentPreviewBubble.swift]: @parth-simformsolutions  is imageTask required here?

File: Sources/StreamChatUI/ChatMessageList/ChatMessage/PhotoCollectionBubble.swift [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+import UIKit
+import StreamChat
+import AVKit
+
+class PhotoCollectionBubble: _TableViewCell {
+
+    // MARK: Variables
+    var content: ChatMessage?
+    var chatChannel: ChatChannel?
+    var isSender = false
```
Comment (ajay-simform) [inline Sources/StreamChatUI/ChatMessageList/ChatMessage/PhotoCollectionBubble.swift]: @parth-simformsolutions  please remove isSender if not required

File: Sources/StreamChatUI/Utils/StackedItems/StackedItem.swift [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    public var attachmentType: AttachmentType?
+
+    public init(id: Int, url: URL, attachmentType: AttachmentType, attachmentId: String? = nil) {
+        self.id = id
+        self.attachmentType = attachmentType
+        self.url = url
+        self.attachmentId = attachmentId
+    }
+
+    public static func staticData() -> [StackedItem] {
```
Comment (ajay-simform) [inline Sources/StreamChatUI/Utils/StackedItems/StackedItem.swift]: @parth-simformsolutions  Please remove this function if it is not required.

**Review by ajay-simform — CHANGES_REQUESTED** [review]

**Review by ajay-simform — APPROVED** [review]

**Pull request merged**